tests/scan/test_render.py

```
@@ -1,11 +1,10 @@
 import unittest
-from unittest import mock
 from unittest.mock import MagicMock, Mock, patch
 from pathlib import Path
 import datetime
 
-from safety.scan.render import print_announcements, print_ignore_details, render_header
-from safety_schemas.models import ProjectModel, IgnoreCodes
+from safety.scan.render import print_announcements, print_summary, render_header
+from safety_schemas.models import ProjectModel, IgnoreCodes, PolicySource
```

Ruff

`safety_schemas.models.IgnoreCodes` imported but unused

Remove unused import: `safety_schemas.models.IgnoreCodes`

(F401)